governance.md

```diff
@@ -0,0 +1,136 @@
+# Main Governance Document
+
+The Project
+===========
+
+The Jupyter/IPython Project (The Project) is an open source software project affiliated with the 501c3 NumFocus Foundation. The goal of The Project is to develop open source software and deploy open and public websites and services for reproducible, exploratory and interactive computing. The Software developed by The Project is released under the BSD (or similar) open source license, developed openly and hosted on public GitHub repositories under the ipython and jupyter GitHub organizations. Examples of Project Software include the IPython Notebook, the IPython Terminal, IPython.parallel, the Jupyter Hub, etc. The Services run by the Project consist of public websites and web-services that are hosted under the jupyter.org or ipython.org domains. The Services Examples of Project Services include the Jupyter and IPython websites ([http://jupyter.org](http://www.google.com/url?q=http%3A%2F%2Fjupyter.org&sa=D&sntz=1&usg=AFQjCNHztfVQq8SN6qni-DVgmoaTYBVasg) and [http://ipython.org](http://www.google.com/url?q=http%3A%2F%2Fipython.org&sa=D&sntz=1&usg=AFQjCNGiroA4bzjI-kCsgV4YdTPbgyuyeA)), nbviewer ([https://nbviewer.ipython.org](https://www.google.com/url?q=https%3A%2F%2Fnbviewer.ipython.org&sa=D&sntz=1&usg=AFQjCNGB-iNba6jJiu1N9Oo4oMTWwFq3Aw)) and the Jupyter coLaboratory ([https://colaboratory.jupyter.org](https://www.google.com/url?q=https%3A%2F%2Fcolaboratory.jupyter.org&sa=D&sntz=1&usg=AFQjCNFBxEodelaV8P877UGqxrp7WU8Y3g)).
```

Comment:
Did you mean to pull in the commits for just this document then? This isn't a full record at all.

```
~/code/src/github.com/ipython/ipython.wiki$ git log IPEP-29\:-Project-Governance.md
commit c99985dbeec1be7b1ecd43ca280d1516e7e80187
Author: Brian E. Granger <ellisonbg@gmail.com>
Date:   Tue Sep 1 22:54:23 2015 -0700

    Silly small stuff.

commit 33f4694dbd1fcd5ea837aa15b934ec87596e07e6
Author: Brian E. Granger <ellisonbg@gmail.com>
Date:   Tue Sep 1 22:54:01 2015 -0700

    Adding note to new jupyter/governance repo.

commit c5ac798ac2e107076360beefff488ce6c6533bc5
Author: ellisonbg <ellisonbg@gmail.com>
Date:   Tue Nov 4 12:38:48 2014 -0800

    Adding Mark Sandler

commit f26042d4070a0c9cb57759709d1094ef0a743a03
Author: fperez <fperez.net@gmail.com>
Date:   Fri Oct 31 10:53:04 2014 -0700

    Uniform terminology for 'non-profit'.

commit 331e06d852d6637466e9012cf9eb0d39def6a3a6
Author: ellisonbg <ellisonbg@gmail.com>
Date:   Mon Oct 27 17:01:40 2014 -0700

    Adding Ins. Partner approval and grace period.

commit 543d133a9e3b5092a8987424d65d565c4f8585cf
Author: ellisonbg <ellisonbg@gmail.com>
Date:   Mon Oct 27 16:56:31 2014 -0700

    Changing BDFL succession process.

commit d00be72bfeda445c8bde738bd4578f9420ba096b
Author: ellisonbg <ellisonbg@gmail.com>
Date:   Mon Oct 27 16:51:09 2014 -0700

    Gold/Silver -> Tier 1/2

commit 59c712083cda75e02a486d91e5f95b1acede9211
Author: takluyver <takowl@gmail.com>
Date:   Fri Oct 24 15:09:24 2014 -0700

    Created IPEP 29: Project Governance (markdown)
```